src/useQuery.js

```diff
@@ -26,7 +26,7 @@ module.exports = function useQuery(query, opts = {}) {
 
   React.useEffect(() => {
     queryReq();
-  }, [query, JSON.stringify(opts.variables)]);
+  }, [query, JSON.stringify(opts)]);
```

Comment:
We could be more selective of the options to resend a query.. for example _should_ we resend the query if `useCache` or `skipCache` changes?

Reply:
Should we pass `allOpts` in here? It would be more correct right? As query and allOpts are what are passed to `useClientRequest`.